src/bot.ts

```diff
@@ -0,0 +1,166 @@
+import { Client, Intents } from "discord.js";
+
+import { schedule } from "node-cron";
+
+import glob from "glob";
+import path from "path";
+
+import { config } from "./config";
+
+import * as decoratorData from "./decorators/data";
+import { makeLogger } from "./logger";
+
+import { CommandRepository } from "./repository/CommandRepository";
+
+const logger = makeLogger(module);
```

Comment:
is this whole file just a rename? confused why every line is marked as new... im not sure what has changed here.

Reply:
the database setup stays in index.ts.
this had to be done because bot.ts imports modules that depend on the DataSource and so there was a circular dependency because the imports tried to import `AppDataSource` before it was defined because it was defined after the imports

Reply:
but bot.ts is 90% of the previous index.ts